src/Place/ReadModel/Duplicate/LookupDuplicatePlaceWithSapi3.php

```diff
@@ -49,6 +49,7 @@ public function getDuplicatePlaceUri(Place $place): ?string
             return $results->getItems()[0]->getUrl()->toString();
         }
 
-        throw new MultipleDuplicatePlacesFound($query);
+        // Add isDuplicate so the response will never contain places we identified as duplicates
+        throw new MultipleDuplicatePlacesFound($query . '&isDuplicate=false');
```

Comment:
I wonder if this extra parameter `isDuplicate=false` also needs to be added to the query we use to get the ID of the duplicate place that was already created.